components/smalltalk/squeak5/Makefile

```diff
@@ -0,0 +1,95 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Package Makefile for the "OpenSmalltalk Cog Spur" Squeak Smalltalk V5 system
+# See http://squeak.org and http://opensmalltalk.org
+#
+
+BUILD_BITS=64
+COMPILER=gcc
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		squeak5
+COMPONENT_VERSION=	5.0.6
+COMPONENT_SUMMARY=	The Squeak V5 Smalltalk System
+COMPONENT_PROJECT_URL=	http://www.squeak.org
```

Comment:
Probably "The Squeak V5 Smalltalk Virtual Machine" fits better.
Maybe you can even think of a better name of the package and its summary if Pharo Smalltalk is also running on this VM?